lib/pharos/salt.rb

```diff
@@ -27,6 +27,11 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Call mine.update on salt for every minion specified by targets.
+    def self.mine_update(targets: "*")
```

Comment:
I think we can use `call` for this directly, and use other keyword arguments, maybe we could narrow the problem down to `Pharos::Salt.call(action: "mine.update")`. By default this method's signature is like: `def self.call(action:, targets: "*", arg: nil)`. It's really close to the `perform` solution that you provide, we don't need to map actions (you'd call the salt action directly), and it's somewhat better to be used from the outside.